src/util/url-filter.js

```diff
@@ -0,0 +1,23 @@
+//Remove unnecessary query params from the URL to avoid redundancy
+const NORMALIZE_OPTS = {
+    'stripWWW': false,
+    'removeDirectoryIndex': false,
+    'removeTrailingSlash': false,
+    'stripFragment': true,
+    'normalizeProtocol': false
+}
+const GSEARCH_SUBSTR = 'www.google'
+const FB_SUBSTR = 'www.facebook'
+const normalizeUrl = require("normalize-url")
```

Comment:
For consistency with the rest of the codebase, use the standard ES6 imports. So this would become 
```js
import normalizeUrl from 'normalize-url'
```